src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -206,6 +206,9 @@
         - name: --enable-private-cluster
           type: string
           short-summary: Enable private cluster.
+        - name: --enable-node-public-ip
+          type: bool
+          short-summary: (PREVIEW) Enable VMSS node public IP.
```

Comment:
Use `is_preview` attribute instead of adding "preview" in plain text.

Reply:
done.